src/lib/s3Uploader.js

```diff
@@ -1,14 +1,30 @@
 import { S3 } from 'aws-sdk';
 
-export const s3 = new S3({
-  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
-  endpoint: process.env.S3_ENDPOINT,
-  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
-  signatureVersion: 'v4',
-  s3ForcePathStyle: true,
-});
+let s3Config;
 
-export const verifyVersioning = async (bucket = process.env.bucket, s3Client = s3) => {
+let bucketName = process.env.S3_BUCKET;
+if (process.env.VCAP_SERVICES) {
+  const { credentials, name } = process.env.VCAP_SERVICES.s3[0];
```

Comment:
Just verified this on cloud.gov using the node REPL. We need to parse the JSON string explicitly

```suggestion
  const { credentials, name } = JSON.parse(process.env.VCAP_SERVICES).s3[0];
```

Or to cleanup the next steps:

```suggestion
  const { s3: [ { credentials: { access_key_id, fips_endpoint, secret_access_key, bucket } } ] } = JSON.parse(process.env.VCAP_SERVICES);
```